generator/parseXML/ParseXML.py

```diff
@@ -651,6 +689,22 @@ def get_dependency_information(self, node):
                      'prefix': prefix})
 
     def read_language_element(self, node):
+        """
+        Interrogates a <language> node and updates global_variables
+
+        :param node: the <language> node
+        :return: returns nothing
+
+        Example of a <language> node starting element:
+        <language name="SBGN" baseClass="SbgnBase" documentClass="SbgnDocument"
+         prefix="Sbgn" libraryName="libSBGN"
+         annotationElementName="Extension" topLevelElementName="Extension"
+         isPackage="false" uses_ASTNode="true" uses_XMLNode="true">
+
+        The <language> node can also contain nodes of at least the following
+        types: <library_version>, <language_version> and <dependencies>.
+        These are also interrogated in this function.
```

Comment:
These would be better as functions for interrogating each type of node.

Reply:
Do you mean we need to add these as new functions?

Reply:
Yes that would be cleaner

Reply:
I will add a TODO comment to the code noting this, for later action.